tasks/c++/easy/sum.cpp

```diff
@@ -1,3 +1,19 @@
 // C++ - Easy 2
 
 // TODO: Create a Basic C++ Program to Add Two Numbers
+#include<iostream>
+
+int sum(int a, int b);
+int main(void)
+{
+    int num1, num2;
+    std::cout << "Enter two numbers: ";
+    std::cin >> num1, std::cin >> num2;
+    std::cout << sum(num1, num2);
```

Comment:
Code looks good! You can also write the line
```cpp
std::cin >> num1, std::cin >> num2;
```
with just
```cpp
std::cin >> num1 >> num2;
```
to make it a bit shorter, but your solution works too just fine.